tensorflow/compiler/xla/service/gpu/gpu_conv_algorithm_picker.cc

```diff
@@ -118,11 +118,11 @@ std::vector<AlgorithmDesc> GetAlgorithms(CudnnConvKind kind,
   return algorithms;
 }
 
-StatusOr<std::vector<se::dnn::ProfileResult>> GetAlgorithms(
+StatusOr<std::vector<se::dnn::ProfileResult>> GetMIOpenAlgorithms(
```

Comment:
I'm not sure this change makes sense, this function is used for both ROCm and CUDA? Isn't MIOpen ROCm-specific?

Reply:
this version of the `GetAlgorithms` function is used on the ROCM side only, which is why it is being renamed to `GetMIOpenAlgorithms` (to make the ROCm specificness explicit)

Reply:
Makes sense!